CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## UNRELEASED
+
+* Removed strict `NibReusable` protocol conforming in `register` functions.
+    You can now make `Reusable` cell, and `NibLoadable` subclass.
```

Comment:
Please add 2 spaces after your full-stop here (see other entries for inspiration).
Ending a line with two spaces in Markdown allows to go to the next line in the same paragraph (like a `<br>`) in the rendered document.

---

The typical formatting I use in the CHANGELOG is this, where `•` represent a space:
```markdown
* Change description.••
  [@your_github_handle](link_to_your_gh_profile)
  [#PR_Num](link_to_this_pr)
```